x-pack/plugins/maps/public/classes/layers/blended_vector_layer/blended_vector_layer.ts

```diff
@@ -97,52 +98,59 @@ function getClusterStyleDescriptor(
       },
     },
   };
-  documentStyle.getAllStyleProperties().forEach((styleProperty: IStyleProperty) => {
-    const styleName = styleProperty.getStyleName();
-    if (
-      [VECTOR_STYLES.LABEL_TEXT, VECTOR_STYLES.ICON_SIZE].includes(styleName) &&
-      (!styleProperty.isDynamic() || !styleProperty.isComplete())
-    ) {
-      // Do not migrate static label and icon size properties to provide unique cluster styling out of the box
-      return;
-    }
+  documentStyle
```

Comment:
Only diff is `styleProperty: IStyleProperty` to `styleProperty: IStyleProperty<StylePropertyOptions>`. The rest of the changes from `pretty` gone wild